sympy/functions/combinatorial/factorials.py

```diff
@@ -294,7 +294,7 @@ class subfactorial(CombinatorialFunction):
     .. math:: !x = \Gamma(x + 1, -1)/e
 
     which is valid for non-negative integers `x`. The above formula
-    is not very useful incase of non-integers. :math:`\Gamma(x + 1, -1)` is
+    is not very useful incase of non-integers. `\Gamma(x + 1, -1)` is
```

Comment:
I guess it should be "in case"

Reply:
Yeah, I'll try to keep it in mind for the next cleanup (not sure it is worth 10 more core hours...).